src/melee/gm/gmmain.c

```diff
@@ -0,0 +1,201 @@
+#include <dolphin/types.h>
+#include <dolphin/gx/gxtypes.h>
+#include <sysdolphin/baselib/controller.h>
+#include <functions.h>
+
+extern s32 lbl_804D4A08; // debug level
+extern BOOL lbl_804D6B20;
+extern u16 lbl_804D6B30; // debug flags
+
+extern GXRenderModeObj lbl_80401168;
+extern PadLibData HSD_PadLibData;
+extern char lbl_803EA6C8[]; // build timestamp string
+extern s32* seed_ptr;
+extern void lbl_8015FD24();
+extern void lbl_803762C4();
+extern void func_801692E8(int, struct datetime* datetime);
+
+enum {
```

Comment:
These DB levels aren't really unknown, since the menus state them.

See: https://github.com/eigenform/melee-re/blob/master/docs/DEBUG.md